connectors/airwatch/src/main/java/com/vmware/connectors/airwatch/AirWatchController.java

```diff
@@ -78,20 +83,29 @@
 
     private final URI gbBaseUri;
 
+    private final long maxAge;
+    private final TimeUnit unit;
+
     @Autowired
     public AirWatchController(WebClient rest, CardTextAccessor cardTextAccessor,
                               AppConfigService appConfig, String connectorMetadata,
-                              URI gbBaseUri) {
+                              URI gbBaseUri,
+                              @Value("${rootDiscovery.cacheControl.maxAge:1}") long maxAge,
+                              @Value("${rootDiscovery.cacheControl.unit:HOURS}") TimeUnit unit) {
         this.rest = rest;
         this.cardTextAccessor = cardTextAccessor;
         this.appConfig = appConfig;
         this.connectorMetadata = connectorMetadata;
         this.gbBaseUri = gbBaseUri;
+        this.maxAge = maxAge;
+        this.unit = unit;
     }
 
-    @GetMapping(path = "/discovery/metadata.json")
-    public ResponseEntity<String> getmetadata() {
-        return ResponseEntity.ok(connectorMetadata);
+    @GetMapping(path = "/")
+    public ResponseEntity<String> getMetadata(HttpServletRequest request) {
+        return ResponseEntity.ok()
```

Comment:
This is repeated for each connector. I would think that modifying the common root controller instead of removing it would be the way to go here.

Reply:
This is good idea.
Done.